Resources/Prototypes/_Goobstation/Entities/Structures/Machines/material_silo.yml

```
@@ -36,6 +36,9 @@
   - type: MaterialStorage
     ignoreColor: true
   - type: Silo
+  - type: DeviceLinkSink
+    ports:
+      - FillbotAnyItem
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)